jobs/kibana-auth-plugin/config/config.sh.erb

```diff
@@ -0,0 +1,9 @@
+export USE_HTTPS="<%= p("kibana-auth.use_https") %>"
+export CF_API_URL="<%= p("kibana-auth.cloudfoundry.api_endpoint") %>"
+export KIBANA_OAUTH2_CLIENT_ID="<%= p("kibana-auth.cloudfoundry.skip_ssl_validation") %>"
+export KIBANA_OAUTH2_CLIENT_SECRET="<%= p("kibana-auth.cloudfoundry.client_id") %>"
```

Comment:
Looks like these properties need to be corrected and I don't see the skip SSL validation property being used here.